applications/CableNetApplication/custom_elements/sliding_cable_element_3D.cpp

```diff
@@ -749,6 +779,10 @@ int SlidingCableElement3D::Check(const ProcessInfo& rCurrentProcessInfo)
     const double domain_size = this->GetCurrentLength();
     KRATOS_ERROR_IF( domain_size <= 0.0 ) << "Element " << this->Id() << " has non-positive size " << domain_size << std::endl;
 
+    if (mpConstitutiveLaw != nullptr) {
```

Comment:
remove the check here, it is enough to check it once in `Check`

Reply:
?

Reply:
I want to call the constitutive check function. Should I do this without 
`if (mpConstitutiveLaw != nullptr)`

Reply:
yes, move this check after the check for existance of the CLaw then you don't have to check twice